config/projects.yaml

```diff
@@ -229,7 +229,14 @@ doctrine-mongodb-admin-bundle:
             php_extensions: ['mongodb-1.9.0', 'zip']
             target_php: '7.4'
             variants:
-                symfony/symfony: ['4.4.*']
+                symfony/symfony: ['4.4.*', '5.3.*']
+                sonata-project/admin-bundle: ['^4@rc']
+        4.x:
+            php: ['7.3', '7.4', '8.0']
+            php_extensions: ['mongodb-1.9.0', 'zip']
+            target_php: '7.4'
```

Comment:
why do you need to restrict this php version?

Reply:
I don't remember if it was because some static analysis issue, I've remove them.

Reply:
Maybe you want to check first on the repo with a sample Pr

Reply:
Apparently they are fine:

https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/633
https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/634